esrally/utils/modules.py

```diff
@@ -63,7 +63,6 @@ def _load_component(self, component_name, module_dirs):
         for p in self._modules(module_dirs, component_name):
             self.logger.debug("Loading module [%s]", p)
             m = importlib.import_module(p)
-            importlib.reload(m)
```

Comment:
this change does not seem to break anything, in testing.  this change was made to support pickling functions for the actor system, as it was breaking the class references after each reload (see a relevant [SO](https://stackoverflow.com/questions/1412787/picklingerror-cant-pickle-class-decimal-decimal-its-not-the-same-object))